### PR TITLE
Document missing Ballerina.toml fields

### PR DESCRIPTION
**Pull request by gayaldassanayake**
## Purpose
$ subject
> Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/8963

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md [hunk 1]
```diff
@@ -195,6 +195,76 @@ version = "0.1.0"
...
+
+```toml
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+license = ["Apache-2.0"]
+```
+
+### The `template` field
```
Comment (azinneera) [inline swan-lake/development-tutorials/source-code-dependencies/package-references.md:254]: Let's remove this. We haven't announced this yet.

File: swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ $ tree .
 2 directories, 5 files
 ```
 
-* The `Ballerina.toml` file identifies the directory as a Ballerina package. You can edit the `Ballerina.toml` file to change the organization, name, and the version of the package. 
+* The `Ballerina.toml` file identifies the directory as a Ballerina package. You can edit the `Ballerina.toml` file to change the organization, name, and the version of the package. To learn about different fields in `Ballerina.toml`, see [The Ballerina.toml file](/learn/package-references/#the-ballerinatoml-file)
```
Comment (azinneera) [inline swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md:36]:
```suggestion
* The `Ballerina.toml` file identifies the directory as a Ballerina package. You can edit the `Ballerina.toml` file to change the organization, name, and version of the package.  Additionally, you have the option to add an icon and a description to your package, as well as keywords allowing users to easily find your package through search functions. To learn more about these fields, see [The Ballerina.toml file](/learn/package-references/#the-ballerinatoml-file)
```

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md [hunk 3]
```diff
@@ -195,6 +195,76 @@ version = "0.1.0"
...
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+keywords = ["service", "edi", "manufacturing"]
+```
+
+### The `authors` field
+
+The `authors` field is used to specify the authors who contributed to the package. This information will be listed under the `Contributors` section in the Ballerina Central API documentation.
```
Comment (azinneera) [inline swan-lake/development-tutorials/source-code-dependencies/package-references.md:214]:
```suggestion
The `authors` field is used to specify the authors who contributed to the package. This information will be listed under the `Contributors` section on the API documentation page in the Ballerina Central. 
```

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md [hunk 4]
```diff
@@ -195,6 +195,76 @@ version = "0.1.0"
 include = ["documents", "sample.png"]
 ```
 
+### The `keywords` field
+
+The keywords field is used to specify a list of short phrases that describe the package. Keywords are helpful for users to discover the package in the Ballerina Central using the built-in search. These keywords are also listed under the `Keywords` section in the Ballerina Central API documentation.
```
Comment (azinneera) [inline swan-lake/development-tutorials/source-code-dependencies/package-references.md:200]:
```suggestion
The keywords field is used to specify a list of short phrases that describe the package. Keywords are helpful for users to discover the package in the Ballerina Central using the built-in search. These keywords are also listed under the `Keywords` section on the API documentation page in Ballerina Central.
```

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md (hunk 3 above)
Comment (azinneera) [reply swan-lake/development-tutorials/source-code-dependencies/package-references.md:214]: Change other places as well.

File: swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md (hunk 2 above)
Comment (gayaldassanayake) [reply swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md]: Fixed with 407ba4be27d26bd90a0bc1938b40f615f289a1ec

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md (hunk 1 above)
Comment (gayaldassanayake) [reply swan-lake/development-tutorials/source-code-dependencies/package-references.md]: Fixed with 407ba4be27d26bd90a0bc1938b40f615f289a1ec

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md (hunk 3 above)
Comment (gayaldassanayake) [reply swan-lake/development-tutorials/source-code-dependencies/package-references.md]: Fixed with 407ba4be27d26bd90a0bc1938b40f615f289a1ec

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md (hunk 4 above)
Comment (gayaldassanayake) [reply swan-lake/development-tutorials/source-code-dependencies/package-references.md]: Fixed with 407ba4be27d26bd90a0bc1938b40f615f289a1ec

**Review by azinneera — APPROVED** [review]

**Review by sm1990 — APPROVED** [review]

**Pull request merged**